panic: dataurl: invalid mediatype

I'm trying to run a replicate image locally with cog 0.11.6:
```
$ cog predict r8.im/thomasmol/whisper-diarization@sha256:cbd15da9f839c5f932742f86ce7def3a03c22e2b4171d42823e83e314547003f \
   -i file=@mnot_short.mp3 \
   -i 'prompt="podcast"' \
   -i 'language="nl"' \
   -i 'translate=true' \
   -i 'num_speakers=1' \
   -i 'group_segments=true' \
   -i 'offset_seconds=0' \
   -i 'transcript_output_format="both"'
```
which runs into an error "panic: dataurl: invalid mediatype":

```
Starting Docker image r8.im/thomasmol/whisper-diarization@sha256:cbd15da9f839c5f932742f86ce7def3a03c22e2b4171d42823e83e314547003f and running setup()...
Running prediction...
panic: dataurl: invalid mediatype

goroutine 1 [running]:
github.com/vincent-petithory/dataurl.New({0xc00073a000, 0x7568a, 0x7568b}, {0x0?, 0xc0003df560?}, {0x0, 0x0, 0x110?})
	/home/runner/go/pkg/mod/github.com/vincent-petithory/dataurl@v1.0.0/dataurl.go:81 +0x28b
github.com/replicate/cog/pkg/predict.fileToDataURL({0x7ffd348dc470, 0xe})
	/home/runner/work/cog/cog/pkg/predict/input.go:106 +0x179
github.com/replicate/cog/pkg/predict.(*Inputs).toMap(0xc00030d7c0)
	/home/runner/work/cog/cog/pkg/predict/input.go:66 +0x106
github.com/replicate/cog/pkg/predict.(*Predictor).Predict(0xc00030d900, 0xc0003c4e70)
	/home/runner/work/cog/cog/pkg/predict/predictor.go:140 +0x5e
github.com/replicate/cog/pkg/cli.predictIndividualInputs({{{0x0, 0x0, 0x0}, {0xc000386540, 0x1, 0x1}, {0xa99fb1, 0x3}, {0x7ffd348dc3fb, 0x6b}, ...}, ...}, ...)
	/home/runner/work/cog/cog/pkg/cli/predict.go:210 +0x2da
github.com/replicate/cog/pkg/cli.cmdPredict(0xc000052d00?, {0xc00018bb00?, 0x4?, 0xa9a229?})
	/home/runner/work/cog/cog/pkg/cli/predict.go:173 +0xb2b
github.com/spf13/cobra.(*Command).execute(0xc0001ed508, {0xc00018b9e0, 0x11, 0x11})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:985 +0xaaa
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001ec608)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x3ff
github.com/spf13/cobra.(*Command).Execute(0xc00020bf40?)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041 +0x13
main.main()
	/home/runner/work/cog/cog/cmd/cog/cog.go:14 +0x66
```

I get the same results with .wav and .mkv files, and I've also tried other mp3 files to make sure it was not a bad audio file or ffmpeg conversion mistake. I've attached a [zipped mp3 file](https://github.com/user-attachments/files/17557114/audio_file_for_testing.zip) for reproducing the issue.

To reproduce:
```
wget https://github.com/user-attachments/files/17557114/audio_file_for_testing.zip
unzip audio_file_for_testing.zip
cog predict r8.im/thomasmol/whisper-diarization@sha256:cbd15da9f839c5f932742f86ce7def3a03c22e2b4171d42823e83e314547003f \
   -i file=@mnot_short.mp3 \
   -i 'prompt="podcast"' \
   -i 'language="nl"' \
   -i 'translate=true' \
   -i 'num_speakers=1' \
   -i 'group_segments=true' \
   -i 'offset_seconds=0' \
   -i 'transcript_output_format="both"'
```

I'm also pretty sure the error is due to cog and not the [underlying Python code](https://github.com/thomasmol/cog-whisper-diarization/blob/main/predict.py). If I pull the image, do `cog run bash` and run the Python code manually from inside the docker container, everything works fine. 

Also note this is different from https://github.com/replicate/cog/issues/1264.

--- a/pkg/predict/input.go
+++ b/pkg/predict/input.go
@@ -2,13 +2,14 @@ package predict
 
 import (
 	"fmt"
-	"mime"
 	"os"
 	"path/filepath"
 	"strings"
 
 	"github.com/mitchellh/go-homedir"
 	"github.com/vincent-petithory/dataurl"
+
+	"github.com/replicate/cog/pkg/util/mime"
 )
 
 type Input struct {

